libtransmission/torrent-files.cc

```diff
@@ -359,6 +364,9 @@ namespace
 // except for the following:
 [[nodiscard]] auto constexpr isReservedChar(char ch) noexcept
 {
+#if !defined(_WIN32)
```

Comment:
```suggestion
#ifndef _WIN32
```

Reply:
I've included this change in #6187, so it's a good advice, but not a blocker for merging this PR. (@ckerr)